components/PartsList/index.js

```diff
@@ -30,31 +32,86 @@ export default function PartsList({ parts }) {
     <>
       <FilterContainer>
         {/* render a filterButton for each object in 'sets' with given 'function', 'value to set' and 'name' */}
+        {/* funct2 + value2 for activeFilter-highlighting */}
         <StyledFilter
           sets={[
-            { funct: setCategoryFilter, value: "alle", name: "alle" },
-            { funct: setCategoryFilter, value: "teller", name: "teller" },
-            { funct: setCategoryFilter, value: "vase", name: "vase" },
-            { funct: setCategoryFilter, value: "schüssel", name: "schüssel" },
-            { funct: setCategoryFilter, value: "figur", name: "figur" },
+            {
+              funct: setCategoryFilter,
+              value: "alle",
+              name: "alle",
+              funct2: setActiveCategoryFilter,
+              value2: "alle",
+            },
+            {
+              funct: setCategoryFilter,
+              value: "teller",
+              name: "teller",
+              funct2: setActiveCategoryFilter,
+              value2: "teller",
+            },
+            {
+              funct: setCategoryFilter,
+              value: "vase",
+              name: "vase",
+              funct2: setActiveCategoryFilter,
+              value2: "vase",
+            },
+            {
+              funct: setCategoryFilter,
+              value: "schüssel",
+              name: "schüssel",
+              funct2: setActiveCategoryFilter,
+              value2: "schüssel",
+            },
+            {
+              funct: setCategoryFilter,
+              value: "figur",
+              name: "figur",
+              funct2: setActiveCategoryFilter,
+              value2: "figur",
+            },
           ]}
+          activeFilter={activeCategoryFilter}
         />
         <StyledFilter
           sets={[
```

Comment:
A smart idea to populate your filters! 🚀 
I suggest some naming changes to shed some light on it for external reviewers (and for yourself in the future when you review your code again after some time has passed 😉 )

```js
{
    primaryFunction: setStatusFilter,
    primaryValue: "alle",
    filterName: "alle",
    secondaryFunction: setActiveStatusFilter,
    secondaryValue: "alle",
}
```

Reply:
Thanks for your advise. I will stay consistent in naming from now on ✌🏼   
it looks muss better now